src/components/Person.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => {
+  const partner = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      <p className="Person__age">
+        {age && `I am ${age}`}
+      </p>
```

Comment:
```suggestion
      {age && (
        <p className="Person__age">
          I am {age}
        </p>
      )}
```

Noe you will render an empty `p` if there is no `age`. Let's not render it at all

Reply:
thank you for noticing this mistake. i have fixed it already